### PR TITLE
Remove links to nonexistent mapfiles

### PR DESCRIPTION
**Pull request by maoznir**

### CODE REVIEW

File: js/canvas-to-blob.min.js [hunk 1]
```diff
@@ -1,2 +1 @@
...
-//# sourceMappingURL=canvas-to-blob.min.js.map
```
Comment (const-cloudinary) [inline js/canvas-to-blob.min.js:2]: No newline at end of file

File: js/load-image.all.min.js [hunk 2]
```diff
@@ -1,2 +1 @@
...
-//# sourceMappingURL=load-image.all.min.js.map
```
Comment (const-cloudinary) [inline js/load-image.all.min.js:2]: No newline at end of file

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

**Review by patrick-tolosa — APPROVED** [review]

**Comment by const-cloudinary** [comment]
@patrick-tolosa @maoznir This fix works with Django 4, we can merge it

File: js/load-image.all.min.js (hunk 2 above)
Comment (maoznir) [reply js/load-image.all.min.js:2]: There is actually a newline, try editing the file and you'll see, I don't know why diff on github shows it like this

File: js/canvas-to-blob.min.js (hunk 1 above)
Comment (maoznir) [reply js/canvas-to-blob.min.js:2]: There is actually a newline, try editing the file and you'll see, I don't know why diff on github shows it like this

**Pull request merged**